Fix build patient purge changes

lmckenzi opened the pull request:
## HL7 FHIR Pull Request

_Note: No pull requests will be accepted against `./source` unless logged in the_ [HL7 Jira issue tracker](https://jira.hl7.org/projects/FHIR/issues/).

If you made changes to any files within `./source` please indicate the Jira tracker number this pull request is associated with: `   `

## Description

_Please describe your pull request here._

grahamegrieve commented:
I don't get this bit: "(in this case, built automatically by the publication tooling)" - what is built automatically? the response, or the content being deleted? Why would a reader care?

lmckenzi commented:
I'd copied that file from the patient, so was just repeating the text.  However, the comment was also wrong, so I've fixed it and dropped that part of the comment because I don't know what it means either.

grahamegrieve reviewed (approved):
ok
